File: ProgrammingFundamentals/Java/04.MethodsDebuggingTroubleshootingCode/Lab/src/P05_TemperatureConversion.java
import java.util.Scanner;
public class P05_TemperatureConversion {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);
        double fahrenheit = Double.parseDouble(console.nextLine());
        double celsius = fahrenheitToCelsius(fahrenheit);
        System.out.format("%.2f", celsius);
    }

    static double fahrenheitToCelsius(double fahrenheit) {
        double result = (fahrenheit - 32) * 5 / 9;
        return result;
    }
}

File: ProgrammingFundamentals/C#/04.MethodsDebuggingAndTroubleshootingCode/Lab/P05_TemperatureConversion/P05_TemperatureConversion.cs
using System;

namespace P05_TemperatureConversion
{
    class P05_TemperatureConversion
    {
        static void Main(string[] args)
        {   
            double fahrenheit = double.Parse(Console.ReadLine());
            double celsius = FahrenheitToCelsius(fahrenheit);
            Console.WriteLine($"{celsius:F2}");
        }

        static double FahrenheitToCelsius(double fahrenheit)
        {
            double result = (fahrenheit - 32) * 5 / 9;
            return result;
        }
    }
}
